README.md

```diff
@@ -10,976 +10,22 @@ allows using docker containers for functional/integration/~~unit~~ testing.
-     assert(pgContainer.jdbcUrl.nonEmpty)
-  }
-}
-```
-
-##### Example with multiple containers
-```scala
-class MySpec extends AnyFlatSpec with TestContainersForAll {
-
-  override type Containers = MySQLContainer and PostgreSQLContainer
-
-  override def startContainers(): Containers = {
-    val container1 = MySQLContainer.Def().start()
-    val container2 = PostgreSQLContainer.Def().start()
-
-    container1 and container2
-  }
-
-  override def afterContainersStart(containers: Containers): Unit = {
-    super.afterContainersStart(containers)
-
-    containers match {
-      case mySqlContainer and pgContainer => println("your logic here")
-    }
-  }
-
-  override def beforeContainersStop(containers: Containers): Unit = {
-    super.beforeContainersStop(containers)
-
-    containers match {
-      case mySqlContainer and pgContainer => println("your logic here")
-    }
-  }
-
-  it should "work" in withContainers {
-    case mysqlContainer and pgContainer =>
-      assert(mysqlContainer.jdbcUrl.nonEmpty && pgContainer.jdbcUrl.nonEmpty)
-  }
-
-}
-```
-
-#### Using fixtures
-
-Instead of the `*ForAll`/`*ForEach` traits you can use the fixtures under `TestContainersFixture`:
-
-```scala
-class MysqlSpec extends FunSuite with TestContainersFixtures {
-
-  // Use `ForAllContainerFixture` to start/stop container before/after all tests
-  val mysql = ForEachContainerFixture(MySQLContainer())
-
-  // You need to override `munitFixtures` and pass in your container fixture
-  override def munitFixtures = List(mysql)
-
-  test("test case name") {
-    // Inside your test body you can do with your container whatever you want to
-    assert(mysql().jdbcUrl.nonEmpty)
-  }
-}
-```
-
-There is also available a `FunFixture` version for containers:
-
-```scala
-class MysqlSpec extends FunSuite with TestContainersFixtures {
-
-  val mysql = ContainerFunFixture(MySQLContainer())
-
-  mysql.test("test case name") { container =>
-    // Inside your test body you can do with your container whatever you want to
-    assert(container.jdbcUrl.nonEmpty)
-  }
-}
-```
-
-#### Notes on MUnit usage
-- If you use `*ForAll` trait and override beforeAll() without calling super.beforeAll() your containers won't start.
-- If you use `*ForAll` trait and override afterAll() without calling super.afterAll() your containers won't stop.
-- If you use `*ForEach` trait and override beforeEach() without calling super.beforeEach() your containers won't start.
-- If you use `*ForEach` trait and override afterEach() without calling super.afterEach() your containers won't stop.
-- [Currently,](https://github.com/scalameta/munit/issues/119) there is no way to retrieve test status in MUnit `afterEach` block, so `afterTest` hook will never contain an error. 
-
-If you have any questions or difficulties feel free to ask it in our [slack channel](https://testcontainers.slack.com/messages/CAFK4GL85).
+* [technical guide](docs/src/main/tut/index.md)
```

Comment:
I think it'd be a bit more discoverable if we move usage one level up and separate docs and the article. Something like this:

---

## Documentation

* [Quick start](https://github.com/sv3ndk/testcontainers-scala/blob/239_update_documentation/docs/src/main/tut/setup.md)
*    [Usage and Examples](https://github.com/sv3ndk/testcontainers-scala/blob/239_update_documentation/docs/src/main/tut/usage.md)

## Links

* [testcontainers-scala in action (blog post)](http://dimafeng.com/2016/08/01/testcontainers-selenium)

Reply:
Good idea, I'll do that. I will be simpler too